Parse/src/main/java/com/parse/ParseCloudCodeController.java

```diff
@@ -46,11 +45,7 @@ public T then(Task<JSONObject> task) throws Exception {
   /* package for test */ Object convertCloudResponse(Object result) {
     if (result instanceof JSONObject) {
       JSONObject jsonResult = (JSONObject)result;
-      try {
-        result = jsonResult.get("result");
-      } catch (JSONException e) {
-        return result;
-      }
```

Comment:
`jsonResult.get("result")` would throw a `JSONException`, which caused this function to return `{}` instead of `null`